requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
-apache-airflow[aws]==1.10.9
-flake8==3.7.9
+apache-airflow[aws]==1.10.10

```

Comment:
May be we can get rid of the `[aws]` requirement since we reverted to not using S3 hook